index.js

```diff
@@ -0,0 +1 @@
+import server from './src/server.ts';
```

Comment:
Why is this file here?

Reply:
It should not. But the lib folder is ignored by git. Probably it is there because of one of the previous compiles. I will make sure that the lib folder includes the latest compile in the next commit.